src/pubsub/ua_pubsub_networkmessage.c

```diff
@@ -892,6 +892,12 @@ UA_NetworkMessage_decodeFooters(const UA_ByteString *src, size_t *offset, UA_Net
 UA_StatusCode
 UA_NetworkMessage_decodeBinary(const UA_ByteString *src, size_t *offset,
                                UA_NetworkMessage* dst) {
+    return UA_NetworkMessage_decodeBinary_custom(src, offset, dst, NULL);
+}
+
+UA_StatusCode
+UA_NetworkMessage_decodeBinary_custom(const UA_ByteString *src, size_t *offset,
+                               UA_NetworkMessage* dst, const UA_DataTypeArray *customTypes) {
```

Comment:
I added a new function that is called by the original `UA_NetworkMessage_decodeBinary` function but with the `UA_DataTypeArray` set to NULL. 

If you are using a custom data type in a network message, you can use the new function `UA_NetworkMessage_decodeBinary_custom` and provide your `UA_DataTypeArray`.